ui/raidboss/data/03-hw/trial/zurvan-ex.ts

```diff
@@ -0,0 +1,234 @@
+import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  flameTarget?: string;
+  tetherBuddy?: string;
+  infiniteElement?: 'fire' | 'ice';
+}
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'ContainmentBayZ1T9Extreme,',
+  zoneId: ZoneId.ContainmentBayZ1T9Extreme,
+  timelineFile: 'zurvan-ex.txt',
+  timelineTriggers: [
+    {
+      id: 'ZurvanEX Metal Cutter',
+      regex: /Metal Cutter/,
+      beforeSeconds: 4,
+      suppressSeconds: 10,
```

Comment:
```suggestion
      suppressSeconds: 16,
```

Hmm.  There's a couple of places with four in a row that seems very noisy to call out multiple tank cleave calls with no mechanic in between.  16 seconds make the first phase a little weird (since maybe you want a call for the first one in the second set), but overall I think is better?

A compromise could be to add a counter and do 10 seconds for the first one 16 seconds suppress after that.

Reply:
I guess maybe we could phase-track and suppress for 10 during the opener and then 16 after The Purge? That would be easily doable.

Reply:
Yeah either way, either phase track or a counter?